integration-tests/load/go.mod

```diff
@@ -626,3 +627,5 @@ exclude github.com/hashicorp/consul v1.2.1
 
 // gotron-sdk is not longer maintained
 replace github.com/fbsobreira/gotron-sdk => github.com/smartcontractkit/chainlink-tron/relayer/gotron-sdk v0.0.5-0.20250422175525-b7575d96bd4d
+
+replace github.com/smartcontractkit/chainlink-evm/gethwrappers => github.com/smartcontractkit/chainlink-evm/gethwrappers v0.0.0-20250808121824-2c3544aab8f3
```

Comment:
what's the reason for that replace? and why only for this `go.mod`?

Reply:
It's not only in this go.mod, it's in any that use gethwrappers. Without the replace it does not work. If you have a suggestion to properly fix it, I'm happy to implement it. This seems to work and unblocks chainlink-evm bumps which was my first prio. I fully agree this isn't the long term version.

Reply:
> It's not only in this go.mod, it's in any that use gethwrappers. Without the replace it does not work.

But that cascade is why it is problematic. What is the root source of this requirement?